test/dash-playlist-loader.test.js

```diff
@@ -397,3 +397,41 @@ QUnit.test('media playlists "refresh" by re-parsing master xml', function(assert
 
   assert.equal(refreshes, 1, 'refreshed playlist after last segment duration');
 });
+
+QUnit.test('delays load when on final rendition', function(assert) {
+  let loader = new DashPlaylistLoader('dash.mpd', this.fakeHls);
+  let loadedplaylistEvents = 0;
+
+  loader.on('loadedplaylist', () => loadedplaylistEvents++);
+
+  // do an initial load to start the loader
+  loader.load();
+  standardXHRResponse(this.requests.shift());
+
+  assert.equal(loadedplaylistEvents, 2, 'two loadedplaylist events after first load');
```

Comment:
might be useful to have a comment here explaining where the 2 loadedplaylist events come from